Makefile

```diff
@@ -332,5 +332,5 @@ $(STACKER):
 	chmod +x $@
 
 .PHONY: test-annotations
-test-annotations: binary check-skopeo $(BATS) $(STACKER)
+test-annotations: binary check-skopeo $(BATS) $(STACKER) $(NOTATION)
```

Comment:
How does cosign get installed? Don't we need to add that as a dependency as well?

Reply:
Yes, I realized that was the missing part. Just added it in the Makefile